pyproject.toml

```diff
@@ -67,3 +67,16 @@ line_length = 88
 indent = 4
 include_trailing_comma = true
 known_first_party = "chemiscope"
+
+### ======================================================================== ###
+
+[project.optional-dependencies]
+explore = [
+    "dscribe",
+    "scikit-learn",
+]
+
+mace = [
+    "scikit-learn",
+    "mace-torch"
+]
```

Comment:
```suggestion
```

I think we decided we did not want other dependencies in chemiscope than the minimal ones

Reply:
Yes, sure, they are only installed with the [tox.docs] right now.
Where should I add those dependencies then? To `docs/requirements` ?

Reply:
yes!